src/panels/Scene/hooks.ts

```diff
@@ -9,25 +10,27 @@ import { useWindowLayoutProvider } from '@/windowmanagement/WindowLayout/hooks';
 export const DefaultSgnWindowId = 'defaultSceneGraphNodeWindow';
 
 export function useOpenCurrentSceneNodeWindow() {
+  const { t } = useTranslation('panel-scene', { keyPrefix: 'hooks' });
   const { ref, addWindow, closeWindow, createWindowTabData } = useWindowLayoutProvider();
 
   function openCurrentNodeWindow(content: React.JSX.Element) {
     if (!ref || ref.current === null) {
-      throw new Error('WindowLayoutProvider ref is not set');
+      throw new Error('WindowLayoutProvider ref is not set'); // TODO: Localize?
```

Comment:
TODO: Decide what to do about errors are remove this TODO comment

Reply:
I'd say to make the distinction of whether the message is actionable to the user.  If the user can do something about it, then it should be translated;  if it is not, then it doesn't need to be translated.

Reply:
I interpret this as not translating the thrown error messages, then. Especially in this case, where the error message is related to the code and not something that the user has access to